contracts/src/access/control.rs

```diff
@@ -182,12 +182,12 @@ impl AccessControl {
     ///
     /// # Errors
     ///
-    /// If [`msg::sender`] has not been granted `role`, then the error
-    /// [`Error::UnauthorizedAccount`] is returned.
+    /// * [`Error::UnauthorizedAccount`] - If [`msg::sender`] has not been
+    ///   granted `role`.
     ///
     /// # Events
     ///
-    /// May emit a [`RoleGranted`] event.
+    /// * [`RoleGranted`].
```

Comment:
```suggestion
    /// * [`RoleGranted`]
```
I think we should omit the dot everywhere if there's no description.
![image](https://github.com/user-attachments/assets/ecbb8c63-c5c7-4c2f-970d-4e38ec06e541)

Reply:
Sounds reasonable, good point! I do not use VS Code but without `.` looks much better.

Reply:
It will look similar in any editor. And even just looking at the docs code, the dot is redundant